scss/_patterns_notifications.scss

```diff
@@ -158,6 +160,12 @@ $notification-text-margin-bottom: $spv--large - $spv-nudge;
       right: $sph--large * 0.5;
       top: $spv--small;
       width: $default-icon-size;
+
+      // FIXME: temporary workaround until icons have proper theme support
+      // stylelint-disable-next-line selector-max-type -- base styles can use type selectors
+      body.is-dark & {
```

Comment:
I think you missed adding the option when notification has `is-dark` class name to this one.